bitsail-common/src/main/java/com/bytedance/bitsail/common/type/filemapping/JdbcTypeInfoConverter.java

```diff
@@ -21,11 +21,18 @@
 import com.bytedance.bitsail.common.typeinfo.ListTypeInfo;
 import com.bytedance.bitsail.common.typeinfo.MapTypeInfo;
 import com.bytedance.bitsail.common.typeinfo.TypeInfo;
+import com.bytedance.bitsail.common.util.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
 
 /**
  * Created 2022/5/11
  */
 public class JdbcTypeInfoConverter extends FileMappingTypeInfoConverter {
+  private static final Logger LOG = LoggerFactory.getLogger(JdbcTypeInfoConverter.class);
```

Comment:
maybe we can add some test for it

Reply:
good idea! I will add test for this class and make checkstyle right